packages/bridge/src/imports/module.ts

```diff
@@ -114,7 +115,7 @@ export default defineNuxtModule<Partial<ImportsOptions>>({
   }
 })
 
-function addDeclarationTemplates (ctx: Unimport) {
+function addDeclarationTemplates (ctx: Unimport, options: ImportsOptions) {
```

Comment:
this doesn't seem to be used

```suggestion
function addDeclarationTemplates (ctx: Unimport) {
```